### ISSUE TITLE
Proofing Agent: Case ID Search

### ISSUE THREAD

**Issue by porta-antiporta** [department-of-veterans-affairs/va.gov-team#66272]
# Use Case

As a PA I am instructed to ask for the enrollment code from the Veteran so that I know how to start the IPP process.

# Context

This is the start of the proofing agent flow after they have successfully authenticated. This story will focus on the happy path where the Veteran has come to them and they do have a valid case ID. 

One thing to keep in mind as we develop this application: unlike Veteran facing services, *the proofing agent does not have any expectations of privacy* we will log as much information as we can about the proofing agent's actions throughout this process. 

However, *the Veteran still has expectations of privacy*. As a result, any information we want to store regarding the Veteran should be minimized and anonymous (i.e. we won't store PII).

![Search case ID](https://github.com/department-of-veterans-affairs/va.gov-team/assets/16668007/2f58d47d-9243-4aa8-98ac-028f2035081d)

# Precondition

- Assume the proofing agent has been authenticated by the time they arrive at this screen. i..e we have access to their Login.gov UUID and x.509 attributes for auditing purposes.

# Flow
1. Proofing agent (PA) asks the Veteran for Case ID. 
2. PA enters the case ID into input field
3. Application searches for case ID in its database of case ID / UUID / email matches.
4. (This is happy path) Application finds the case ID.

# Acceptance criteria
Scenario: A Proofing Agent is searching for a Case Number for a Veteran
GIVEN the Veteran has a Case Number
WHEN the Proofing Agent enters it into the field
THEN the Proofing Agent can click "Continue"

Questions/Notes:
- "Continue" not active (cannot proceed) until Case number input
- will we allow special characters to be input? Na for now.
- Out of Scope: what will error message look like if Case Number is not found? Suggest trying again? Confirm number and reenter? 

#Def of Done
- [ ] Page rendered as designed
- [ ] Accept only a specific hardcoded Case ID (for demo purposes)
